Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL maintainers="pavel@wikiteq.com,alexey@wikiteq.com"
 LABEL org.opencontainers.image.source=https://github.com/WikiTeq/Taqasta
 
 ENV MW_VERSION=REL1_39 \
-	MW_CORE_VERSION=1.39.8 \
+	MW_CORE_VERSION=1.39.10 \
```

Comment:
it seems that the 1.39.10 update was added to this branch too - please leave that to #223 and don't change it here

Reply:
I put note https://github.com/WikiTeq/Taqasta/pull/223